### PR TITLE
UI overhaul.

### PR DESCRIPTION
**Pull request by gdiazc**
bitcoin-sentiments.js:
- Don't use DataTable
- Eliminate scatterplot

index.html:
- Reorder columns,
- Simplify tweet table
- Update jQuery and Bootstrap sources

app.py:
- Remove scatter plot

plotting.py:
- Nicer plot

### CODE REVIEW

File: static/bitcoin_sentiments.js [hunk 1]
```diff
@@ -18,21 +18,42 @@ angular.module('BitcoinSentimentsApp', [])
...
+            tweetTable.tweets = [
+                {'id_str': 1, 'full_text': 'working!'}
+            ]
+
+            BitcoinSentimentsService.getTweetsFromServer()
+            .then(function(response) {
+                return response.json()
+            }).then(function(data) {
+                $('#plot_card_spinner').css('display', 'none')
+                Bokeh.embed.embed_item(data['plot2'], 'mainplot_div')
```
Comment (dev1-fund3) [inline static/bitcoin_sentiments.js]: Include timezone for the time axis label.

File: es_client.py [hunk 2]
```diff
@@ -55,23 +55,23 @@ def get_tweets(client, max_tweets=5000):
...
+        .query('match_all')\
+        .sort({'created_at': {'order': 'desc'}})[:max_tweets]
     search.execute()
 
     tweets = []
-    for row in search.scan():
-        if len(tweets) <= max_tweets:
-            created_at = _get_with_default(row, 'created_at')
-            full_text = _get_with_default(row, 'full_text')
+    for hit in search:
```
Comment (dev1-fund3) [inline es_client.py:60]: Use list splicing to simplify to `for hit in search[:max_tweets]:` and remove if statement below.

**Review by dev1-fund3 — CHANGES_REQUESTED** [review]
UI changes look great! Clarification on timezone for axis required and small optimization in for loop.

File: static/bitcoin_sentiments.js (hunk 1 above)
Comment (gdiazc) [reply static/bitcoin_sentiments.js]: Fixed in [01c0714](https://github.com/fund3/sentiment-dashboard/pull/4/commits/01c0714ebbc5e4495f9523b09bb94b9ea89d1b7a).

**Review by dev1-fund3 — APPROVED** [review]
Looks good. Let's merge!

**Pull request merged**